frontends/web/src/routes/buy/info.tsx

```diff
@@ -151,12 +151,12 @@ class BuyInfo extends Component<Props, State> {
                                                 </tr>
                                                 <tr>
                                                     <td>{t('buy.info.disclaimer.payment.table.2_method')}</td>
-                                                    <td class={style.nowrap}>5.9 %</td>
+                                                    <td class={style.nowrap}>4.9 %</td>
                                                     <td>{t('buy.info.disclaimer.payment.table.2_description')}</td>
                                                 </tr>
                                                 <tr>
                                                     <td>{t('buy.info.disclaimer.payment.table.3_method')}</td>
```

Comment:
This is apple/google/samsung pay, right. Did we try and know it is actually working?

Reply:
No worries, it was for the future. Remember next time :)